coffee-shop-kata-solutions/pom.xml

```diff
@@ -34,7 +34,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
-        <junit5.version>5.10.0</junit5.version>
```

Comment:
The Surefire plugin version needs to be upgraded in this POM too

Reply:
Fixed